content/en/opentelemetry/interoperability/instrumentation_libraries.md

```diff
@@ -24,13 +24,26 @@ This provides observability for libraries not covered by Datadog SDKs without ch
 
 Before adding OpenTelemetry instrumentation libraries, set the `DD_TRACE_OTEL_ENABLED` environment variable to `true`.
 
+Datadog SDKs provide an implementation of the OpenTelemetry API that submits spans to the Datadog Agent using Datadog Span Exporters. To use the OpenTelemetry
+Instramentations with Datadog SDKs ensure a Datadog Agent is configured.
+
 <div class="alert alert-warning">
 When replacing a Datadog instrumentation with its OpenTelemetry equivalent, disable the
 Datadog instrumentation to avoid duplicate spans in the trace.
 </div>
 
+<div class="alert alert-warning">
+Datadog SDKs implement the OpenTelemetry API by overriding the default implementations in the OpenTelemetry SDK. Using operations only supported by 
+OpenTelemetry SDK are not supported (ex: SpanProcessors, OTLP Trace Exporters).
+</div>
+
+<div class="alert alert-warning">
+Datadog SDKs do not support OpenTelemetry Metrics and Logs APIs. Support for Metrics and Logs will be introduced in future releases. To use OpenTelemetry Logs and Metrics apis
```

Comment:
In a previous PR, I noticed the docs team say that we don't "document the future". You may want to remove the second sentence here and simply redirect to our OTLP ingest method